Moved to script folder

rolandomunoz opened the pull request:
- Moved to script folder

Commit:
Moved to script folder

jeanphilippegoldman reviewed (approved):
please modify the menu as well

Commit:
- Fixed: Do not show a crash window when there are no objects in the Object window
- Changed script path

rolandomunoz commented:
Done!

Commit:
Changes summary:
	- Changed script syntax
	- Moved to script folder
	- Fixed a bug when using the command `Save as short text file...`

Changes to be committed:
	deleted:    save_files2.praat
	new file:   scripts/save_files2.praat
	modified:   setup.praat

Commit:
Commit Summary:
	Moved to scripts folder
	Do not call setbasename.praat temporally

Changes to be committed:
	renamed:    get_durations.praat -> scripts/get_durations.praat
	modified:   setup.praat

Commit:
Commit Summary:
	Update to new syntax
	Moved to script folder
	Do not call setbasename.praat temporally

 Changes to be committed:
	deleted:    resample_files.praat
	new file:   scripts/resample_files.praat
	modified:   setup.praat
